Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -5357,6 +5444,24 @@
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
+		7B9FC39428A26137007570E7 /* TestIPC */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 7B9FC57B28A26137007570E7 /* Build configuration list for PBXNativeTarget "TestIPC" */;
+			buildPhases = (
+				7B9FC39B28A26137007570E7 /* Sources */,
+				7B9FC3A328A26137007570E7 /* Frameworks */,
+				7B9FC5B428A52224007570E7 /* Embed Frameworks */,
```

Comment:
Does TestIPC really need an "Embed Frameworks" phase? It's not a bundle with a frameworks directory, and it would normally be executed alongside WebCore.framework and others.